staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -718,10 +724,18 @@ func patchResource(
 			if err := applyPatchToObject(codec, currentObjMap, originalPatchMap, versionedObjToUpdate, versionedObj); err != nil {
 				return nil, err
 			}
+			// Convert the object back to unversioned.
 			objToUpdate := patcher.New()
-			if err := convertor.Convert(versionedObjToUpdate, objToUpdate, nil); err != nil {
+			gvks, _, err = typer.ObjectKinds(objToUpdate)
+			if err != nil {
+				return nil, err
+			}
+			gvk = gvks[0]
+			unversionedObjToUpdate, err := unsafeConvertor.ConvertToVersion(versionedObjToUpdate, gvk.GroupVersion())
```

Comment:
I don't understand it - it doesn't compile for me.
[]GVK doesn't implement GroupVersioner.